crates/biome_js_semantic/src/events.rs

```diff
@@ -502,6 +503,7 @@ impl SemanticEventExtractor {
         if is_exported {
             self.stash.push_back(SemanticEvent::Exported {
                 range: node.syntax().text_range(),
+                declaration_start: node.syntax().text_range().start(),
```

Comment:
I thought `range` and `declaration_start` belonged to two different nodes, but it's not. Why do we need `declaration_start`, when we can compute it from `range`?